packages/node_modules/@ciscospark/widget-space/src/actions.js

```diff
@@ -92,14 +102,21 @@ function fetchingSpaceDetails() {
 /**
  * Gets details about the space
  * @param {Object} sparkInstance
- * @param {String} spaceId
+ * @param {String} destinationId
+ * @param {String} destinationType
  * @param {Object} intl
  * @returns {Thunk}
  */
-export function getSpaceDetails(sparkInstance, spaceId, intl) {
+export function getSpaceDetails(sparkInstance, destinationId, destinationType, intl) {
```

Comment:
Should we move the options into a single object instead of adding more params to this function?